src/lib.rs

```diff
@@ -69,7 +69,9 @@ mod fp12;
 mod fp6;
 
 // The BLS parameter x for BLS12-381 is -0xd201000000010000
+#[allow(dead_code)]
```

Comment:
Probably instead of that, add the feature that requires this constants. Since it's not dead code, is instead only used behind some features.

Reply:
Fixed